.github/workflows/generate-catalog.yaml

```diff
@@ -0,0 +1,174 @@
+# MIT License
+# Copyright (c) 2021 Eli Lilly and Company
+# Author: William Michael Landau (will.landau at gmail)
```

Comment:
I would like to avoid dependent licenses if possible?

Reply:
I'm not sure what you mean by dependent licenses? That would suggest R packages dependencies would also need to be avoided 😬